packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -25,7 +30,8 @@ function useGlobalStylesRenderer() {
 		}
 
 		const currentStoreSettings = getSettings();
-		const nonGlobalStyles = currentStoreSettings?.styles?.filter(
+		const nonGlobalStyles = filter(
+			currentStoreSettings.styles,
```

Comment:
Same reason for using lodash `filter`. Can handle both `array|object` and will return an empty array when styles are undefined. Optional chaining returns `undefined` and  makes `nonGlobalStyles` non-iterable.